sig-cli/CONTRIBUTING.md

```diff
@@ -206,6 +183,8 @@ A feature request is an [issue] mentioning `@kubernetes/sig-cli-feature-requests
 To encourage readership, the issue description should _concisely_ (2-4 sentence) describe
 the problem that the feature addresses.
 
+
+//Absolutelly not sure about these 2 next sections. Feel super heavyweight though. Is this the way the SIG is working actually? 
 ### Feature lifecycle
```

Comment:
I don't think the design proposal process is what is used anymore.  This is handled more informally during meetings, and then results in a KEP for something that needs a design.

I am not familiar with "design proposals" in the context of Kubernetes, so maybe someone else who has more background can clarify if this should be removed or updated.